sources/Metal/Renderers/iTermCursorGuideRenderer.m

```diff
@@ -126,37 +166,59 @@ - (void)setColor:(NSColor *)color {
 - (void)drawWithFrameData:(iTermMetalFrameData *)frameData
            transientState:(__kindof iTermMetalCellRendererTransientState *)transientState {
     iTermCursorGuideRendererTransientState *tState = transientState;
-    if (tState.row < 0 || tState.column < 0) {
+    if (tState.row < 0 && tState.column < 0) {
         return;
     }
 
-    [tState initializeVerticesWithPool:_cellRenderer.verticesPool];
-    if (tState.row >= 0 && self.horizontalEnabled) {
+    if (tState.row >= 0 && tState.column >= 0 && self.horizontalEnabled && self.verticalEnabled) {
+        [tState initializeVerticesWithPool:_cellRenderer.verticesPool horizontal:TRUE vertical:TRUE];
```

Comment:
Use YES and NO instead of TRUE and FALSE. Also on line 197 and 206.

Reply:
Done.